### DISCUSSION TITLE
Select checkbox can not  unselect filter items only

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#6252] [Help]
Hi All,

I tried to use the checkbox to unselect the filtered items only  but it did not work.  Unchecked will unselect all items.

Not sure how to make it works.  See screenshot.

![image](https://user-images.githubusercontent.com/44040606/152063579-3d2074e2-de9b-486a-bf1f-b2bc4a88619b.png)

**Comment by wezmag** [comment]
You can refer to the source code of [GridRowSelectionMixin](https://github.com/serenity-is/Serenity/blob/a1a0bda9ba15f7b981cefbe13e2552bd07ac8e39/src/Serenity.Scripts/CoreLib/UI/Helpers/SlickHelpers.ts#L20), and write your own version. 
[Line# 62-64](https://github.com/serenity-is/Serenity/blob/a1a0bda9ba15f7b981cefbe13e2552bd07ac8e39/src/Serenity.Scripts/CoreLib/UI/Helpers/SlickHelpers.ts#L62-L64) is where that needs to be changed.
